controllers/trainees/trainees.controller.ts

```diff
@@ -128,6 +136,101 @@ class TraineesController {
 			console.log(e);
 		}
 	}
+
+	static async editProfile(req: Request, res: Response) {
+		const {
+			userId,
+			firstName,
+			lastName,
+			githubUsername,
+			tel,
+			bio,
+			education,
+			targetWorkCity,
+			workExperience,
+			portfolioUrl1,
+			portfolioUrl2,
+			portfolioUrl3,
+			portfolioUrl4,
+			portfolioUrl5,
+			projectUrl1,
+			projectUrl2,
+			projectUrl3,
+			projectUrl4,
+			projectUrl5,
+			course1,
```

Comment:
this should be one string field - courses. And delete course2, course3, course4, course5.